packs/feat-effects/effect-turn-aside-ambient-magic.json

```
@@ -0,0 +1,69 @@
+{
+    "_id": "uRwnbZ0xWrB5ZzZ9",
+    "img": "systems/pf2e/icons/default-icons/effect.svg",
+    "name": "Effect: Turn Aside Ambient Magic",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Turn Aside Ambient Magic]</p>\n<p><span style=\"font-family:Roboto, sans-serif\">Your ostilli glows a faint yellow as it establishes a barrier. Choose acid, cold, electricity, fire, or sonic damage. Until the beginning of your next turn, you gain resistance against the chosen damage type equal to half your level.</span></p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Turn Aside Ambient Magic]</p>\n<p>Your ostilli glows a faint yellow as it establishes a barrier. Choose acid, cold, electricity, fire, or sonic damage. Until the beginning of your next turn, you gain resistance against the chosen damage type equal to half your level.</p>"
```
Careful of the cruft that comes from pasting directly into the editor. This is why I prefer to only work directly with the HTML.